src/libs/OptionsListUtils.js

```diff
@@ -443,8 +443,14 @@ function getOptions(reports, personalDetails, activeReportID, {
             return;
         }
 
-        // We let Free Plan default rooms to be shown in the App - it's the one exception to the beta, otherwise do not show policy rooms in product
-        if (ReportUtils.isDefaultRoom(report) && !Permissions.canUseDefaultRooms(betas) && ReportUtils.getPolicyType(report, policies) !== CONST.POLICY.TYPE.FREE) {
+        // We create policy rooms for all policies, however we don't show them unless
+        // - It's a free plan workspace
+        // - The report includes guides participants (@team.expensify.com) for 1:1 Assigned
+        if (ReportUtils.isDefaultRoom(report)
+            && !ReportUtils.hasExpensifyGuidesEmails(logins)
```

Comment:
Very very very minor nab, but this check is in a different order than the others